example/lib/account_overview_screen.dart

```diff
@@ -62,6 +63,43 @@ class AccountOverviewScreenState extends State<AccountOverviewScreen> {
     fetchBalance();
   }
 
+  void testZkSyncTx() async {
```

Comment:
Do we want to fully commit this to our example app given it's with a custom client config? Or should we just include it in the code level docs, and then add in support later once we've more formally created the network config for it?